README.md

```diff
@@ -32,6 +32,13 @@ If you want tests to automatically rerun when you change files, run
 tests when relevant files change. To run all tests, just hit enter in the
 guard interactive console when you see the `>` prompt.
 
+## Linting
+
+This project uses [rubocop](https://www.rubocop.org/en/stable/) to lint ruby code.
+To run ruby linting, call `docker-compose (exec or run) web bin/lint.sh --ruby`.
```

Comment:
You can say `run`.  Honestly, my instructions for testing should say `run` too.  Exec doesn't work if the container isn't already up.

Reply:
See #61; I figured that it was worth mentioning that both work, but adding jibber-jabber about it in this pr was kinda out of scope.

Reply:
Yes, but I like simple copy-pasteable commands in the readme, so we should just tell them to use `run`.

Reply:
Ok, fixed